engine/api/test/test.go

```diff
@@ -85,7 +85,7 @@ func SetupPG(t log.Logger, bootstrapFunc ...Bootstrapf) (*gorp.DbMap, cache.Stor
 	if err != nil {
 		t.Fatalf("Unable to connect to redis: %v", err)
 	}
-	event.Cache = store
+	event.Initialize(event.KafkaConfig{}, store)
```

Comment:
Error return value of `event.Initialize` is not checked

Reply:
but tests...